css/types/imagetypes.json

```
@@ -0,0 +1,385 @@
+                      },
+                      "safari": {
+                        "version_added": true
+                      },
+                      "safari_ios": {
+                        "version_added": true
+                      },
+                      "samsunginternet_android": {
+                        "version_added": true
+                      },
+                      "webview_android": {
+                        "version_added": true
+                      }
+                    },
+                    "status": {
+                      "experimental": false,
+                      "standard_track": true,
+                      "deprecated": false
+                    }
+                  },
+                  "element": {
+                    "__compat": {
+                      "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/element",
+                      "description": "<code>element()</code>",
+                      "support": {
+                        "chrome": {
+                          "version_added": false
+                        },
+                        "chrome_android": {
+                          "version_added": null
+                        },
+                        "edge": {
+                          "version_added": null
+                        },
+                        "edge_mobile": {
+                          "version_added": null
+                        },
+                        "firefox": {
+                          "prefix": "-moz-",
+                          "version_added": "4"
+                        },
+                        "firefox_android": {
+                          "prefix": "-moz-",
+                          "version_added": "4"
+                        },
+                        "ie": {
+                          "version_added": false
+                        },
+                        "opera": {
+                          "version_added": false
+                        },
+                        "opera_android": {
+                          "version_added": false
+                        },
+                        "safari": {
+                          "version_added": false
+                        },
+                        "safari_ios": {
+                          "version_added": false
+                        },
+                        "samsunginternet_android": {
+                          "version_added": false
+                        },
+                        "webview_android": {
+                          "version_added": false
+                        }
+                      },
+                      "status": {
+                        "experimental": true,
+                        "standard_track": true,
+                        "deprecated": false
+                      }
+                    }
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+}
```

This needs a new line at the end:
```suggestion
}
```